runner/execer/execer.go

```diff
@@ -46,7 +46,7 @@ func (s ProcessState) String() string {
 }
 
 func (s ProcessState) IsDone() bool {
-	return s == COMPLETE || s == FAILED
+	return s != RUNNING
```

Comment:
what about PENDING or other states?

Reply:
That doesn't exist in the execer, all we have is UNKNOWN, RUNNING, COMPLETE, and FAILED